samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -85,6 +85,16 @@ spec:
           sentinel: sentinel
           redisReplicas: 5
 
+        - name: jwt
```

Comment:
Samples here https://github.com/dell/csm-operator/blob/main/config/samples/storage_v1_csm_authorization_v2.yaml also has to be updated

Reply:
Fixed